lib/server.rb

```diff
@@ -1,10 +1,29 @@
-require "sinatra"
+require "grape"
+require 'grape-swagger'
 
-class Server < Sinatra::Base
-  get "/" do
-    repo = Repositories::ExampleRepository.new
-    res = repo.test_query
+class Server < Grape::API
+  version 'v1', using: :path, vendor: 'forms'
+  format :json
+  prefix :api
+
+  resource :forms do
+    desc 'Returns all forms.'
+    params do
+      requires :result, type: Integer, desc: 'Result ID.'
+    end
+    route_param :result do
+      get do
+        Services::Example.new.execute(params[:result])
+      end
+    end
```

Comment:
@danielburnley calling this out early, like the format of grape so far? Big example looks pretty tasty to me https://github.com/ruby-grape/grape#basic-usage